src/axl.c

```diff
@@ -610,6 +657,74 @@ int AXL_Dispatch (int id)
     return rc;
 }
 
+/*
+ * Given a path with an AXL temporary extension, allocate an return a new
+ * string with the extension removed.  Also, if extra is specified, return
+ * a pointer to the offset in 'path_with_extension' where the 'extra' field
+ * is.
+ *
+ * Returns the new allocated path string on success, or NULL on error.
+ */
+static char * axl_remove_extension(char *path_with_extension, char **extra)
+{
+	int i;
+	size_t size = strlen(path_with_extension);
+    char *ext = AXL_EXTENSION;
+    size_t ext_len = sizeof(AXL_EXTENSION) - 1; /* -1 for '\0' */
```

Comment:
Just missing some spaces here I think
```suggestion
      char *ext = AXL_EXTENSION;
      size_t ext_len = sizeof(AXL_EXTENSION) - 1; /* -1 for '\0' */
```

Reply:
Fixed in latest push